js/src/as-report.js

```diff
@@ -50,7 +50,7 @@ class ActionsReport extends Component {
 			if ( query.status !== prevQuery.status ) {
 				query.orderby = 'scheduled';
 				query.order = 'asc';
-				query.per_page = prevQuery.per_page;
+				query.per_page = prevQuery.per_page ? prevQuery.per_page : 100;
```

Comment:
Just a nitpick: `100` looks like a magic number. It could be improved using a constant named like `DEFAULT_PER_PAGE`.

Reply:
@adrianduffell Thanks for the suggestion. I've made the change and will go ahead and merge this.